osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr422.json

```diff
@@ -412,18 +412,18 @@
         {
           "id": 3,
           "description": "scaler_unit",
-          "note": "Attribute 3 (scaler_unit) is overridden in factory with value -1, V (default in spec: 0, V)",
+          "note": "Attribute 3 (scaler_unit) is default in spec: '0, V', this is overridden in factory with value -1, V for some meter-types (so use valuetype dynamic)",
           "datatype": "scal_unit_type",
-          "valuetype": "FIXED_IN_PROFILE",
-          "value": "-1, V",
+          "valuetype": "DYNAMIC",
```

Comment:
DYNAMIC means the value can be updated by the meter itself. That's not the case here. I would suggest SET_BY_CLIENT or FIXED_IN_METER.